examples/Microsoft.Spark.CSharp.Examples/MachineLearning/Sentiment/README.md

```diff
@@ -0,0 +1,188 @@
+
+As we create the logic for our Spark app, we'll paste in the code generated from Model Builder and include some other class definitions.
+
+## .NET for Spark
+
+Now that we've trained an ML.NET model for sentiment analysis, we can begin writing the .NET for Spark code that will read in our Yelp data, pass each review to our ML.NET model, and predict whether reviews are positive or negative.
+
+### 1. Create a Spark Session
+
+In any Spark application, we need to establish a new SparkSession, which is the entry point to programming Spark with the Dataset and 
+DataFrame API.
+
+```CSharp
+SparkSession spark = SparkSession
+       .Builder()
+       .AppName(".NET for Apache Spark Sentiment Analysis")
+       .GetOrCreate();
+```
+
+### 2. Read Input File into a DataFrame
+
+We trained our model with the Amazon data, so let's test how well the model performs by testing it with the Yelp dataset. 
+
+```CSharp
+DataFrame df = spark.Read().Csv(<Path to yelp data set>);
+```
+
+If we want to specify some other aspects of our data, such as whether it has a header and how we want to deal with its schema, we can set some other options when reading in our data:
+
+```CSharp
+DataFrame df = spark
+    .Read()
+    .Option("header", true)
+    .Option("inferSchema", true)
+    .Csv(<Path to yelp data set>);
+```
+
+### 3. Use a UDF to Access ML.NET
+
+We create a User Defined Function (UDF) that calls the *Sentiment* method on each Yelp review.
+
+```CSharp
+spark.Udf().Register<string, bool>("MLudf", (text) => Sentiment(text));
+```
+
+*Sentiment* is where we'll call our ML.NET code that was generated from the final step of Model Builder.
+
+```CSharp
+MLContext mlContext = new MLContext();
+ITransformer mlModel = mlContext.Model.Load(@"Resources\MLModel.zip", out var modelInputSchema);
+var predEngine = mlContext.Model.CreatePredictionEngine<Review, ReviewPrediction>(mlModel);
+```
+
+You may notice the use of *Review* and *ReviewPrediction.* These are classes we define in our project to represent the review data we're evaluating. 
+
+```CSharp
+public class Review
+{
+      // Represents the input review text
+      [LoadColumn(0)]
+      public string Column1;
+}
+```
+
+```CSharp
+public class ReviewPrediction : Review
+{
+
+       [ColumnName("PredictedLabel")]
+       public bool Prediction { get; set; }
+
+       public float Probability { get; set; }
+
+       public float Score { get; set; }
+} 
+```
+
+### 4. Spark SQL and Running Your Code
+
+Now that you've read in your data and incorporated ML, use Spark SQL to call the UDF that will run sentiment analysis on each row of your DataFrame:
+
+```CSharp
+DataFrame sqlDf = spark.Sql("SELECT _c0, MLudf(_c0) FROM Reviews");
```

Comment:
This is different than what's in the code.

Reply:
Updated it now.